tools/@o3r/test-helpers/scripts/utilities/create-test-app.mjs

```diff
@@ -0,0 +1,113 @@
+import {execSync} from 'node:child_process';
+import {existsSync, mkdirSync, readFileSync, rmSync} from 'node:fs';
+import * as path from 'node:path';
+import {satisfies} from 'semver';
+import {Locker} from './locker.mjs';
+
+/**
+ * Generate a base app with minimal necessary dependencies
+ * Uses a locker mechanism so this function can be called in parallel
+ * The lock will automatically expire after 10 minutes if the creation of the app failed for whatever reason
+ *
+ * @param appName
+ */
+export async function createTestApp(options) {
```

Comment:
Why not TS? As they are actually required in IT that are written in TS?

Reply:
updated with ts files